src/app/admin/organization-page/organization-page.component.spec.ts

```diff
@@ -49,7 +49,7 @@ describe('OrganizationPageComponent', () => {
   it('Display the name of the organization',() => {
     component.organizationId = testData.orgList[index].id;
     fixture.detectChanges();
-    de = fixture.debugElement.query(By.css('div.org-name'));
+    de = fixture.debugElement.query(By.css('h2.org-title'));
```

Comment:
this change makes this test identical to the previous test. you should either remove this test, or display the organization name as a field.